content/authors/misra-turp/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: author
+title: Mısra Turp
+name: Misra Turp
+slug: misra-turp
+position: Developer Educator @AssemblyAI
```

Comment:
Maybe we can start adding '(Guest Author)' - next to the full position?

Reply:
Then the title becomes too long imo
![image](https://github.com/deepset-ai/haystack-home/assets/25897116/384032df-b3cd-4bc2-b6b0-743669f2008d)